yb-voyager/cmd/templates/migration_assessment_report.template

```diff
@@ -673,156 +723,15 @@
                     </tbody>
                 {{ end }}
             </table>
-        {{ else }}
+
+        {{ else if not .onlyPerf }} 
             <p>No issues were found in the assessment.</p>
         {{ end }}
+   
 
-        {{ $performanceOptimizationIssues := getPerformanceOptimizationIssues .Issues }}
```

Comment:
what happens to minimum version fixed in case of perf optimisations? Is that omitted?

Reply:
that won't be relevant for perf optimisations right? Since its a subjective thing.

Reply:
It is possible in the future if DB starts doing some of these directly internally so we might need to call that out.